ansible/playbooks/service/phpipam.yml

```diff
@@ -10,5 +10,38 @@
 
   roles:
 
+    - role: debops.php/env
+      tags: [ 'role::php', 'role::apt_preferences', 'role::logrotate' ]
+
+    - role: debops.apt_preferences
+      tags: [ 'role::apt_preferences', 'skip::apt_preferences' ]
+      apt_preferences__dependent_list:
+        - '{{ php__apt_preferences__dependent_list }}'
+        - '{{ nginx__apt_preferences__dependent_list }}'
+
+    - role: debops.logrotate
+      tags: [ 'role::logrotate' ]
+      logrotate__dependent_config:
+        - '{{ php__logrotate__dependent_config }}'
+
+    - role: debops.php
+      php__dependent_packages:
+        - '{{ phpipam__php__dependent_packages }}'
+      php__dependent_pools: [ '{{ phpipam__php5_pool }}' ]
```

Comment:
The `phpipam__php5_pool` variable should be renamed to `phpipam__php__dependent_pools`. You could also put it on a separate line here, like the rest of the dependent variables.

Reply:
done